docs/resources/Guild.md

```diff
@@ -191,6 +191,7 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
 | premium_since? | ?ISO8601 timestamp                              | when the user used their Nitro boost on the server               |
 | deaf           | boolean                                         | whether the user is deafened in voice channels                   |
 | mute           | boolean                                         | whether the user is muted in voice channels                      |
+| hoisted_role?  | ?snowflake                                      | the highest displayed role that a member has, nulled if they don't have any |
```

Comment:
why is this null and optional? I think this is a mistake, but if it isn't, then we should document the difference between it not being included at all and it being included but being null.

Probably worth using `hoisted` rather than `displayed` for this for consistency, as if you click the user's profile, that displays the roles may or may not be hoisted, so it might be ambiguous (only a nitpick though).

Reply:
I personally have only seen it nulled, but @z64 said "it won't be filled in other contexts", so maybe he can provide which situations it is present and nulled vs which situations it is just not there.

Reply:
Sorry for the delay @A5rocks, I missed this notification.

Objects in the Resources section account for both the HTTP API and the Gateway, with exceptions noted where relevant. In this case, this field will not be present when observing member objects from the HTTP API.

```
~> curl -s \
   -H "Authorization: $TOKEN" \
   https://discordapp.com/api/v6/guilds/602921050824769559/members/120571255635181568 \
   | jq 
{
  "user": {
    "id": "120571255635181568",
    "username": "z64",
    "avatar": "4c136bb9cafe08babc51b1479aceeb8f",
    "discriminator": "1337"
  },
  "roles": [
    "602922354301206528"
  ],
  "nick": null,
  "premium_since": null,
  "mute": false,
  "deaf": false,
  "joined_at": "2019-07-22T17:53:00.401000+00:00"
}